config/knexfile.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     client: 'pg',
     connection: process.env.DATABASE_URL,
     migrations: { directory: '../data/migrations' },
-    seeds: { directory: '../data/seeds' },
+    seeds: { directory: '/c/Users/ajgeb/Desktop/My_Projects/labs/story-squad-be-b/data/seeds' },
```

Comment:
now that we're running raw SQL on the reset-button, we won't need to use this absolute-path. Let's revert back to using the relative-path.